packages/amplify-graphql-transformer-interfaces/src/transform-host-provider.ts

```diff
@@ -31,7 +32,7 @@ export interface DynamoDbDataSourceOptions extends DataSourceOptions {
 export interface TransformHostProvider {
   setAPI(api: GraphqlApiBase): void;
 
-  addHttpDataSource(name: string, endpoint: string, options?: DataSourceOptions, scope?: Construct): HttpDataSource;
+  addHttpDataSource(name: string, endpoint: string, options?: HttpDataSourceOptions, scope?: Construct): HttpDataSource;
```

Comment:
This was necessary to pass the `authorizationConfig` (signing region / service name) to the HTTP data source on creation. [[Docs reference](https://docs.aws.amazon.com/appsync/latest/APIReference/API_AuthorizationConfig.html)]